superset/db_engine_specs/hive.py

```diff
@@ -437,14 +437,9 @@ def get_configuration_for_impersonation(
         url = make_url(uri)
         backend_name = url.get_backend_name()
 
-        # Must be Hive connection, enable impersonation, and set param
+        # Must be Hive connection, enable impersonation, and set optional param
         # auth=LDAP|KERBEROS
-        if (
-            backend_name == "hive"
-            and "auth" in url.query.keys()
-            and impersonate_user is True
-            and username is not None
-        ):
+        if backend_name == "hive" and impersonate_user is True and username is not None:
```

Comment:
Could you also replace `impersonate_user is True` with `impersonate_user`?

Reply:
Sure, done :)